airflow/providers/openlineage/provider.yaml

```diff
@@ -135,3 +135,11 @@ config:
         example: ~
         type: boolean
         version_added: ~
+      dag_state_change_process_pool_size:
+        description: |
+          Number of processes to use for processing DAG state changes.
```

Comment:
It might be helpful to describe what component these processes are being executed on